app/src/main/java/org/mozilla/fenix/components/Core.kt

```diff
@@ -474,6 +476,16 @@ class Core(
         }
     }
 
+    fun getHttpsOnlyMode(): HttpsOnlyMode {
```

Comment:
nit: Just a thought: there already are similar methods in this file that "get" some values for core init, but IMO this would better fit in the Settings file seeing as it returns values that depend only on shared prefs.

Reply:
Thanks, will move.